pyaedt/hfss.py

```diff
@@ -3610,13 +3610,16 @@ def edit_source_from_file(
         is_time_domain : bool, optional
             Either if the input data is Time based or Frequency Based. Frequency based data are Mag/Phase (deg).
```

Comment:
```suggestion
            Whether the input data is time-based or frequency-based. Frequency based data are Mag/Phase (deg).
```
The second sentence here should be rewritten in plain English.